code/modules/clothing/suits/job_suits.dm

```diff
@@ -238,11 +241,11 @@
 	actions_types = list(/datum/action/item_action/button)
 	adjust_flavour = "unbutton"
 
-/obj/item/clothing/suit/storage/lawyer/purpjacket
+/obj/item/clothing/suit/storage/iaa/purplejacket
 	name = "purple suit jacket"
 	desc = "A snappy dress jacket."
-	icon_state = "suitjacket_purp"
-	item_state = "suitjacket_purp"
+	icon_state = "suitjacket_purple"
+	item_state = "suitjacket_purple"
 	blood_overlay_type = "coat"
 	body_parts_covered = UPPER_TORSO|ARMS
```

Comment:
The desc, blood_overlay_type, and body_parts_covered are the same across all of these subtypes, can you move them up to their comment parent type?